src/main/java/org/thoughtcrime/securesms/components/emoji/AutoScaledEmojiTextView.java

```diff
@@ -50,36 +50,34 @@ private float getTextScale(String text) {
     if (text.length() > 21 || text.isEmpty() || Character.isLetter(text.charAt(0))) {
```

Comment:
```suggestion
    if (text.length() > 56 || text.isEmpty() || Character.isLetter(text.charAt(0))) {
```

Reply:
without this, 4-8 emojis of some flags would not be properly scaled

Reply:
See comment above, there is no evidence that this PR here improves performance, so it's no reason to increase the limit

Reply:
the reason to increase the limit is not for performance reasons but for what this PR is targeting that is showing composite emojis enlarged, if I got it correctly this PR was never about performance :D

Reply:
oth, showing big only up to 3 of such emojis is good enough probably